src/Microsoft.Diagnostics.Monitoring.EventPipe/Counters/CounterFilter.cs

```diff
@@ -56,11 +68,13 @@ public bool IsIncluded(string providerName, string counterName)
             {
                 return true;
             }
-            if (_enabledCounters.TryGetValue(providerName, out List<string> enabledCounters))
+            if (_enabledCounters.TryGetValue(providerName, out var enabledCounters))
```

Comment:
I'd recommend we keep an explicit type. I think the usual BCL guideline is not to use 'var' if someone reading just this line code wouldn't be able to infer the type.